src/Tizen.NUI/src/public/WindowSystem/QuickPanelClient.cs

```diff
@@ -0,0 +1,422 @@
+﻿/*
+ * Copyright(c) 2020 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Tizen.Applications.Exceptions;
+
+namespace Tizen.NUI.WindowSystem.Shell
+{
+    /// <summary>
+    /// Class for the Tizen quickpanel client.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
+    public class QuickPanelClient : IDisposable
+    {
+        private TizenShell _tzsh;
+        private IntPtr _tzshQpClient;
+        private int _tzshWin;
+        private bool disposed = false;
+
+        private int _visibleEventType;
+        private IntPtr _visibleEventHandler;
+        private Interop.QuickPanelClient.QuickPanelEventCallback _onVisibleChanged;
+        private event EventHandler<StateVisible> _visibleChanged;
+        /// <summary>
+        /// Emits the event when the visible is changed.
+        /// </summary>
+        /// <exception cref="ArgumentException">Thrown when failed of invalid argument.</exception>
+        /// <since_tizen> 8 </since_tizen>
+        public event EventHandler<StateVisible> VisibleChanged
```

Comment:
VisibilityChanged is normally used

Reply:
In the window system, "visibility" is used for inside of enlightenment.
And using "visible" is used for distinguish with that value.
And "visible" is used in tizen-ws-shell library same.
So fix this event name along may occur confuse.